Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM cvriend/pgs:latest
+WORKDIR /
+
+# Need fslinstaller.py
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y wget bc dc tree parallel zip && rm -rf /var/lib/apt/lists/*
+
+RUN wget https://fsl.fmrib.ox.ac.uk/fsldownloads/fslconda/releases/fslinstaller.py && python fslinstaller.py -d /usr/local/fsl/ -V 6.0.7.13
+
+RUN echo '\n # FSL Setup \nFSLDIR=/usr/local/fsl \nPATH=${FSLDIR}/share/fsl/bin:${PATH} \nexport FSLDIR PATH \n. ${FSLDIR}/etc/fslconf/fsl.sh' >> /root/.bashrc
```

Comment:
not for this PR, but as you mentioned offline, let's think about using installing only conda to install only the necessary FSL packages, perhaps using [miniforge as a base image](https://github.com/conda-forge/miniforge-images/tree/master)